cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -89,6 +89,7 @@
     public static final boolean DEFAULT_NETWORK_POLICY_GENERATION = true;
     public static final boolean DEFAULT_CREATE_CLUSTER_ROLES = false;
     public static final boolean DEFAULT_POD_SET_RECONCILIATION_ONLY = false;
+    public static final String DEFAULT_STRIMZI_OPERATOR_IMAGE = "quay.io/strimzi/operator:latest";
```

Comment:
How is this and the uses of this variable related to this PR?

Reply:
This is just to have a single place for default image URL.

Reply:
Yeah, I understood it. But having a separate PR for it would make it much easier to review things

Reply:
While the change it self might make sense, I think I counted at least 8 files changed only by this. It complicates the review and makes it harder to track changes in the future.

Reply:
OK.